packages/contracts-bedrock/test/setup/ForkLive.s.sol

```diff
@@ -49,20 +49,36 @@ contract ForkLive is Deployer {
 
     /// @notice Forks, upgrades and tests a production network.
     /// @dev This function sets up the system to test by:
-    ///      1. reading the superchain-registry to get the contract addresses we wish to test from that network.
-    ///      2. deploying the updated OPCM and implementations of the contracts.
-    ///      3. upgrading the system using the OPCM.upgrade() function.
+    ///      0. reading the environment variable to determine the base and OP chain names, read the
+    /// SUPERCHAIN_OPS_ALLOCS_PATH data
+    /// created from superchain ops.
```

Comment:
nit on formatting
```suggestion
    ///      0. reading the environment variable to determine the base and OP chain names, read the
    ///         SUPERCHAIN_OPS_ALLOCS_PATH data created from superchain ops.
```

Reply:
Also, what env var are we reading here to "determine the base and OP chain names"? I don't really understand this comment